content/en/docs/tasks/job/pod-failure-policy.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Handling retriable and non-retriable pod failures with Pod failure policy
+content_type: task
+min-kubernetes-server-version: v1.25
+weight: 60
+---
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+<!-- overview -->
+
+This task demonstrates how to use the
+[Pod failure policy](/docs/concepts/workloads/controllers/job#pod-failure-policy),
+in combination with the default
+[Pod backoff failure policy](/docs/concepts/workloads/controllers/job#pod-backoff-failure-policy),
+to improve your control over the handling of container- or Pod-level failure
+within a {{<glossary_tooltip text="Job" term_id="job">}}.
+
+This may help you to better utilize the computational resources by avoiding
```

Comment:
```suggestion
The definition of Pod failure policy and Pod backoff failure policy may help you to better utilize the computational resources by avoiding
```

Reply:
I think connecting both policies by `and` suggest they play the same role here, which might be misleading. Pod failure policy is meant to `fix` limitations of Pod backoff failure policy which is enabled by default.

Reply:
Rephrased, please take a look